htdocs/compta/facture/list.php

```diff
@@ -197,7 +197,7 @@
 	'f.total_localtax2'=>array('label'=>$langs->transcountry("AmountLT2", $mysoc->country_code), 'checked'=>0, 'enabled'=>($mysoc->localtax2_assuj == "1"), 'position'=>120),
 	'f.total_ttc'=>array('label'=>"AmountTTC", 'checked'=>0, 'position'=>130),
 	'u.login'=>array('label'=>"Author", 'checked'=>1, 'position'=>135),
-	'dynamount_payed'=>array('label'=>"Received", 'checked'=>0, 'position'=>140),
+	'dynamount_payed'=>array('label'=>"Payed", 'checked'=>0, 'position'=>140),
```

Comment:
We should not have same label for supplier and customer invoice.
On is amount paid, other is amount received.